File: src/main/java/me/yoursole/ctf/Commands/Start.java
package me.yoursole.ctf.Commands;

import me.yoursole.ctf.DataFiles.GameData;
import me.yoursole.ctf.DataFiles.Items.Flag;
import org.bukkit.Bukkit;
import org.bukkit.ChatColor;
import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;

public class Start implements CommandExecutor {
    @Override
    public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
        if(sender.isOp()){
            Player player = Bukkit.getOnlinePlayers().stream().skip((int) (Bukkit.getOnlinePlayers().size() * Math.random())).findFirst().orElse(null);
            player.getInventory().addItem(Flag.flag);
            GameData.it=player;
            player.sendMessage(ChatColor.GREEN+"You got the flag to start!");
            return true;
        }else{
            sender.sendMessage("don't u dare ._.");
            return true;
        }

    }
}

File: src/main/kotlin/me/yoursole/ctf/commands/Start.kt
package me.yoursole.ctf.commands

import me.yoursole.ctf.data.GameData
import me.yoursole.ctf.data.items.Flag
import org.bukkit.Bukkit
import org.bukkit.ChatColor
import org.bukkit.GameMode
import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender

class Start : CommandExecutor {
    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<String>): Boolean {
        return if (sender.isOp) {
            val player = Bukkit.getOnlinePlayers().filter { it.gameMode == GameMode.SURVIVAL }.random()
            player.inventory.addItem(Flag.flag)
            GameData.it = player
            player.sendMessage("${ChatColor.GREEN}You got the flag to start!")
            true
        } else {
            sender.sendMessage("don't u dare ._.")
            true
        }
    }
}